website/docs/index.html.markdown

```diff
@@ -45,7 +47,9 @@ terraform {
 
 # Configure the Microsoft Azure Provider
 provider "azurerm" {
-  features {}
+  features {
+    skip_provider_registration = true # This is only required when the user running Terraform lacks the permissions to register Azure Resource Providers.
```

Comment:
I think this should be a separate example, and we should keep the basic example also, but update the comment to be explicit in the requirement for any auth method? 
```suggestion
    skip_provider_registration = true # This is only required when the User, Service Principal, or Identity running Terraform lacks the permissions to register Azure Resource Providers.
```